store/src/java/com/zimbra/cs/service/mail/SaveProfileImage.java

```diff
@@ -128,10 +128,12 @@ private static void updateLDAP(long inputSize, InputStream thumbnailIn, String c
                 Provisioning.getInstance().modifyAttrs(mbox.getAccount(), prefs, true,
                     zsc.getAuthToken());
             } else {
-                throw ServiceException.FAILURE("Unable to update thumbnail image in LDAP", null);
+                ZimbraLog.mailbox.warn("Unable to resize profile image");
+                throw ServiceException.FAILURE("FAILURE_MESSAGE", null);
             }
         } else {
-            throw ServiceException.FAILURE("Unable to update thumbnail image in LDAP", null);
+            ZimbraLog.mailbox.warn("Profile image is larger than maximimum allowed size for resizing");
```

Comment:
nits:
1. typo maximimum → maximum
2. given this is a log only message and it sounds quite odd anyway, why not just log specifically what we're comparing it against?
For example:
```"Profile image is larger than localconfig max_image_size_to_resize"```
or
```"Profile image is larger than maximum size allowed (max_image_size_to_resize)"```
We could even log the actual size...
```"Profile image is larger than maximum size allowed (max_image_size_to_resize): %d > %d", inputSize, LC.max_image_size_to_resize.intValue()```

Reply:
Fixed